class.jetpack-onboarding-end-points.php

```diff
@@ -531,12 +533,54 @@ static function install_theme() {
 					wp_send_json_error('Could not install theme (unspecified server error)');
 					die();
 				}
-			} 
+			}
 		}
 
 		wp_send_json_success( $theme_id );
 	}
 
+	static function have_contact_info_widget( $widgets ) {
+		foreach ($widgets as $widget) {
+    		if ( strpos( $widget, 'widget_widget_contact_info' ) ) {
```

Comment:
That check will not be necessary if we ask the user whether they want to insert the widget. This could be done in the business screen, or in a new screen after it, with 2 fields:

* Insert Contact Info widget? (checkbox)
* Sidebar to insert into (dropdown field with all sidebars), shown if the above is checked and there is at least one registered sidebar.